packages/api/src/services/emp-registry.ts

```diff
@@ -22,8 +22,10 @@ export default (config: Config, appState: Dependencies) => {
       endBlock
     );
     const { contracts } = registry.getEventState(events);
-    await Promise.map(Object.keys(contracts || {}), (x) => {
-      return registeredEmps.add(x);
+    await Promise.map(Object.keys(contracts || {}), (address) => {
+      const blockNumber = (contracts as any)[address]["blockNumber"];
```

Comment:
@daywiss I'm not sure how to properly assign a type to the raw event

Reply:
hmm yea im not sure. Maybe you could use objects.entries, which would give you the key and value, but  you might need to still cast the value.  You could also import this type from the sdk and cast to this: https://github.com/UMAprotocol/protocol/blob/master/packages/sdk/src/clients/registry/client.ts#L22

Reply:
This is the ethers event, right?

Is there a central location where we could put the `GetEventType` generic so it could be used in situations like this?

Reply:
the type is exposed in the sdk and can be used directly, which i think would be better than fetching it from `GetEventType`. Would be something like `uma.clients.registry.NewContractRegistered`. Is this what you mean?

Reply:
It looks like I was not able to get the correct type because of some Visual Studio Code issue. I went through all the source code and I knew that the type is set correctly. 
I ran `yarn build` for `sdk` and `api` again, delete Visual Studio cache and now it works.